Docs: adds v0.25.0 Console changelog callouts (#1137)
pomerium/documentation: content/docs/deploy/enterprise/changelog.mdx

# Changelog

## 0.25.0

> ### New

Reviewer: ```suggestion
### Breaking
```

> - In this release, we removed support for the `debug` option in the Console

Reviewer: ```suggestion
- In this release, we removed support for the `debug` option in the Enterprise.
```

> - In this release, we removed support for the `debug` option in the Enterprise.

Reviewer 2: Is "the Enterprise" correct? Should this be "the Enterprise Console"?

Author: This was per @desimone 's edits, but I think Enterprise Console sounds fine.

Author: ```suggestion
- In this release, we removed support for the `debug` option in the Enterprise Console.
```

### Fixes

> - Various UI improvements and an update to the route import feature to support the following PPL criteria when importing routes into the Console:

Reviewer: ```suggestion
- Various UI improvements and an update to the route import feature to support the following PPL criteria when importing routes into Enterprise:
```

  - `allow_public_unauthenticated_access`
  - `allow_any_authenticated_user`
  - `allowed_idp_claims`

### Changed

> - Adds an optional, global-level [Pass Identity Headers](/docs/reference/pass-identity-headers) setting, which sends identity headers to all upstream applications when enabled. If you only want to forward identity headers to a specific upstream application, you can still use the [per-route Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) setting.

Reviewer 2: ```suggestion
- Adds an optional, global-level [Pass Identity Headers](/docs/reference/pass-identity-headers) setting, which sends identity headers to all upstream applications when enabled. If you want to forward identity headers only to a specific upstream application, you can still use the [per-route Pass Identity Headers](/docs/reference/routes/pass-identity-headers-per-route) setting.
```

> - Removes support for the Cookie Secure setting in the Console

Reviewer: ```suggestion
- Removes support for the Cookie Secure setting in Enterprise. 
```

> - Removes support for the Cookie Secure setting in the Console

Reviewer: Cookie Secure 
```suggestion
- Removes support for the [Secure Cookie](https://en.wikipedia.org/wiki/Secure_cookie) setting. It is always enabled by default. 
```
